README.md

```diff
@@ -1,5 +1,10 @@
 # Google Software Product Sprint
 
+This repo contains my work for Google's SPS program (summer 2020).
+
+
+<-- Original Repo -->
```

Comment:
This is not in standard markdown format. (https://www.markdownguide.org/basic-syntax/) I guess you will have future update on this file.